mod/calculated_metrics/set/ltype_rtype/record/year.rb

```diff
@@ -2,10 +2,6 @@
 
 include_set Type::MetricAnswer
 
-def unknown?
-  answer.blank?
-end
-
```

Comment:
not necessary, since `virtual?` is overridden.

I note this in some coming comments, but imo `virtual?` is the only state method that we should ever overwrite.

Reply:
I guess it's possible that there could be a case where there is a real card but the lookup answer is blank.  But that would be a data problem.

Reply:
It was already common practice to override virtual? when I did this. I don't remember the details but I think there was a reason that I did it this way. 
Could be to keep the edit menu because all these cards have editable content. Maybe that no longer necessary because the old menu logic is gone.